doc/connectivity/resting_state_networks.rst

```diff
@@ -25,76 +24,66 @@ Multi-subject ICA: CanICA
       fMRI datasets", `NeuroImage Vol 51 (2010)
       <http://www.sciencedirect.com/science/article/pii/S1053811910001618>`_, p. 288-299
 
-Data preparation: retrieving example data
------------------------------------------
+Objective
+----------
+ICA is a useful approach for finding independent sources of fMRI
+timeseries covariance without needing to define any priors. ICA
+and similar techniques can be therefore used to define regions or
+networks that share similar BOLD signal across time. The CanICA
+incorporates information both within-subjects and across subjects
+to arrive at consensus components.
 
-We will use down-sampled data from the `brain development dataset
-<https://osf.io/5hju4/files/>`_ has been
-preprocessed using `FMRIPrep and Nilearn <https://osf.io/wjtyq/>`_.
-We use nilearn
-functions to fetch data from Internet and get the filenames (:ref:`more
-on data loading <loading_data>`):
+We can demonstrate this technique using down-sampled data from the 
+`brain development dataset <https://osf.io/5hju4/files/>`_, which
+has been preprocessed using `FMRIPrep and Nilearn <https://osf.io/wjtyq/>`_.
```

Comment:
Shouldn't this be in the example, rather than the narrative documentation. Here is what I suggest:

"""
.. topic:: Nilearn data for examples

   Nilearn provides easy-to-analyze data to explore functional connectivity and resting: the 
   `brain development dataset <https://osf.io/5hju4/files/>`_, which
   has been preprocessed using `FMRIPrep and Nilearn <https://osf.io/wjtyq/>`_.

"""

And the following two lines also in the topic block.

Reply:
Yeah I agree, I went back and forth about removing this. Just to clarify your suggestion here -- did you want to move this to the example, or keep it in the User Guide here but change the formatting per your present suggestions?